src/createServer.js

```diff
@@ -1,10 +1,47 @@
+/* eslint-disable max-len */
 'use strict';
 
+/* eslint-disable node/no-deprecated-api */
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    const url = new URL(req.url, `http://${req.headers.host}`);
+    const absolutePaths = path.resolve(url.pathname);
```

Comment:
The use of `path.resolve(url.pathname)` is incorrect for checking if the path is absolute. `path.resolve` resolves a sequence of paths or path segments into an absolute path, which is not the intended use here. Consider using `path.isAbsolute` to check if a path is absolute.